docs/reference/release-notes.md

```diff
@@ -0,0 +1,33 @@
+# COS 2 release notes
+October 2025
+
+These release notes cover new features and changes in COS 2.
+
+COS 2 is designated as a short-term support release. This means it will continue to receive security updates and critical bug fixes for 9 months.
+
+See our [release policy](release-policy.md) and [upgrade instructions](../how-to/upgrade.md).
+
+## Requirements and compatibility
+See [system requirements](system-requirements.md).
+
+COS 2 is compatible with Juju v3.6+.
+
+## What's new in COS 2
+
+- Terraform modules for COS and COS Lite.
+- Grafana v12. We upgraded the workload version from Grafana 9 to Grafana 12. A thorough review of Grafana's breaking changes and how they affect us is available [on Discourse](https://discourse.charmhub.io/t/cos-will-start-using-grafana-12-what-changed/18868).
+- `extra_alert_labels` config option. A new config option in grafana-agent and opentelemetry-collector enabled adding custom labels to alert rules. This is useful for differentiating alerts from site with different SLAs.
```

Comment:
"This is useful for differentiating alerts from site with different SLAs." - is this supposed to be from "sites" (plural)?  It reads a bit off to me, but I'm not sure if that was the issue.

"This" in "This is useful" is a bit ambiguous - it can be interpreted as saying the config option is useful for XYZ or that the labels themselves are useful for XYZ. It's sort of the same thing when you think about it, but then the user has to stop and think about it :shrug: Changing it to "This option is useful..." or "Custom labels are useful..." (or similar) is a little easier to read

Reply:
Done.